easybuild/easyconfigs/p/preseq/preseq-2.0.2-foss-2016b.eb

```diff
@@ -0,0 +1,29 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'preseq'
+version = '2.0.2'
+
+homepage = 'http://smithlabresearch.org/software/%(name)s'
+description = """Software for predicting library complexity and genome coverage in
+ high-throughput sequencing."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/smithlabcode/%(name)s/releases/download/v%(version)s']
+sources = ['%(name)s_v%(version)s.tar.bz2']
+
+checksums = ['1d7ea249bf4e5826e09697256643e6a2473bc302cd455f31d4eb34c23c10b97c']
+
```

Comment:
more style nitpicking: drop empty line above

Reply:
@boegel I changed this.